src/overlays/actors/ovl_En_Bigpo/z_en_bigpo.c

```diff
@@ -1354,73 +1332,64 @@ void func_80B6467C(Actor *thisx, GlobalContext *globalCtx) {
 }
 
 // draw func
-// non-equivelent: this is an odd draw function, 
-// diff is nowhere close
 // set as poe is dying
-#ifdef NON_EQUIVELENT
 void func_80B64880(Actor *thisx, GlobalContext *globalCtx) {
     EnBigpo* this = (EnBigpo*) thisx;
-
-    Camera *cam;
     f32 magnitude;
     f32 magnitude2;
-    Vec3f vec1; // sp64
+    Gfx* dispHead;
+    Vec3f vec1;
     Vec3f vec2;
-    Gfx *dispHead;
+    Camera* cam = globalCtx->cameraPtrs[globalCtx->activeCamera];
 
-    OPEN_DISPS(globalCtx->state.gfxCtx);
-
-    cam = globalCtx->cameraPtrs[globalCtx->activeCamera];
     if (cam != NULL) {
         Math_Vec3f_Diff(&cam->eye, &cam->focalPoint, &vec1);
         magnitude = Math3D_Vec3fMagnitude(&vec1);
-        //magnitude2 = (magnitude > 1.0f) ? (20.0f / magnitude) : (20.0f);
-        //Math_Vec3f_Scale(&vec1, magnitude2);
-        Math_Vec3f_Scale(&vec1, (magnitude > 1.0f) ? (20.0f / magnitude) : (20.0f));
+        magnitude2 = (magnitude > 1.0f) ? (20.0f / magnitude) : (20.0f);
+        Math_Vec3f_Scale(&vec1, magnitude2);
```

Comment:
Just an fyi, both `cam` and `magnitude2` are temps that aren't needed/can be replaced with pad without changing the asm. This is important for finding the match that involved `OPEN_DISPS`, you'll need to free at least 1 temp minimum to bring that back. Possibly need the second temp too to find whatever match is out there